src/types/db.ts

```diff
@@ -0,0 +1,3 @@
+export interface TilesRow {
+    tile_data: object;
+}
```

Comment:
Is this file the closest thing we have to a db schema?
It looks like the db stores tilesrow objects, which each contain a single column called 'tile_data'.
What is the format of a 'tile_data' object? I assume there is a lower-level breakdown than _row_, so we can get specific tiles that live on the row.

Reply:
NB this file might mean 'tile row' in the sense of 'database row' but I read it in the sense of 'row of tiles in the 2D grid'

Reply:
This interface is really just there to keep Typescript happy. 
In the db, there's a view called `tiles` which we're selecting from with zoom_level, tile_column and tile_row INTEGER columns (our query targets these) and tile_data BLOB column (what we return - the data for the single tile we want). 
This interface was indeed intended to mean "database results row". Will rename, but will also see if we can do something better around surfacing the SQL schema. Might just be a case of better commenting at least!